src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
- [ ] great job on the project 👍  in order to make your project look even more professional please make sure you add a descriptive readme file, you can use the template provided by Microverse under [this link](https://github.com/microverseinc/readme-template).